pages/edge/learn/connecting-domains-to-edge.mdx

```diff
@@ -0,0 +1,65 @@
+import { Callout } from "nextra-theme-docs";
+
+# Connecting your domain to Wasmer Edge
+
+## Introduction
+
+For any functional website/server, you need a domain name. This is the address that people will use to access your services.
+For example, the domain name for this website is `wasmer.io` and the domain for our registry is `registry.wasmer.io`.
+
+In this guide, we will show you how to connect your domain to Wasmer Edge.
+
+## Subdomain
+
+If you want to connect a subdomain(e.g. `blogs.wasmer.io`) to your app hosted on Wasmer Edge, you will need to create a CNAME record pointing to your app on wasmer edge.
+
+```yaml
+blogs.wasmer.io. 0 3600 IN CNAME wasmer-blogs.wasmer.app.
+```
+
+And it should be done! You can now access your app on `blogs.wasmer.io`.
+
+## Root domain
+
+Connecting a root domain is a bit more complicated than connecting a subdomain because you can't create a CNAME record for a root domain. But there are some providers that allow you to create a CNAME record for a root domain. If your provider doesn't allow you to do that, you can use a service like [Cloudflare](https://www.cloudflare.com/) to create a CNAME record for your root domain.
+
+<Callout type="default" emoji="⚠️">
+  If your domain is not controlled by cloudflare you can set it up a transfer by
+  following the guide{" "}
+  <a href="https://developers.cloudflare.com/registrar/get-started/transfer-domain-to-cloudflare/">
+    here
+  </a>
+</Callout>
+
+<Callout type="info" emoji="ℹ️">
+  Cloudflare does this by **CNAME flattening**. You can read more about it{" "}
+  <a href="https://blog.cloudflare.com/introducing-cname-flattening-rfc-compliant-cnames-at-a-domains-root/">
+    here
+  </a>
+</Callout>
+
+If you want to connect your root domain (e.g. `wasmer.io`), to your app hosted on Wasmer Edge
+You will need to create two additional records:
+
+    1. A CNAME record pointing to your app on wasmer edge.
+    2. A TXT record with the value `cname: wasmer.sh.       cname:wasmer--wasmer-sh.wasmer.app."
+
+```yaml
+wasmer.io.              0       IN      CNAME   wasmer-website.wasmer.app
+wasmer.io.              0       IN      TXT     "cname:wasmer-website.wasmer.app."
+```
+
+And it should work now. You can now access your app on `wasmer.io`.
+
+Let's breakdown the configuration:
+
+1. The first record is a CNAME record pointing to your app on wasmer edge.
+2. The second record is a TXT record which Wasmer Edge uses to verify that you own the domain and it needs to connect to your app.
+
+## Future improvements
+
+We are working on making this process easier. We are planning to add features such as:
+
+1. Adding custom edge nameservers to your domain.
```

Comment:
The below is a bit inaccurate and probably confusing for users.
Let's just drop this and stick to :

> We are working on making this process easier in the future. Follow the Edge feature announcements for updates!